### ISSUE TITLE
Add Go.Tag annotation support

### ISSUE THREAD

**Issue by mjcowan** [thriftrw/thriftrw-go#293]
Please add go.tag annotation support for fields.

It would be great if all tags could be converted to go tags i.e

```
struct Foo {
   1: optional string baz (tag1 = 'foobar')
}
```

Generates

```
type Foo struct {
   baz *string `tag1="foobar"`
}
```

**Comment by Raynos** [comment]
Partially implemented by #305

**Comment by alextbok** [comment]
Implemented via https://github.com/thriftrw/thriftrw-go/pull/315 with the caveat that only `go.tag` annotations are converted to go tags
